### DISCUSSION TITLE
Disappearing window border on workspace change

### DISCUSSION THREAD

**Discussion by andynameistaken** [asmvik/yabai#1379] [Q&A] [answered]
HI, is there any option to move window from one workspace to another with preserving border around it?
I have this problem since `4.0.0v`.

**Comment by asmvik** [comment, marked as answer]
Sounds like a bug, https://github.com/koekeishiya/yabai/issues/1229

You can toggle the window border off and back on using `yabai -m window --toggle border` as a workaround until it gets fixed.
